### DISCUSSION TITLE
Joint procurement - Authority & Entity Activity when mixing contracting CA/CE parties

### DISCUSSION THREAD

**Discussion by DeanFirkelj** [OP-TED/eForms-SDK#953] [Q&A] [answered]
Hi,

We are currently running v1.10 and there seems to be a recent change in the version which has some unusual combination of validations.
The case is: Utilites Directive, Multiple Contracting authorities, one of which is a Contracting authority (PartyTypeCode => body-pl).
 Validations BR-BT-00610-0023 & BR-BT-00010-0023 in combination require:

- That the contracting authority as Contracting party must have authority-activity (which is completely logical)
- That the contracting authority as Contracting party must have entity-activity

At the end, we managed to successfully validate the notice by submitting BOTH authority-activity & entity-activity for the CA.
Previously the validations forced only sending the entity-activity in this case. 
I'm not sure whether this is by design, or this will change - so please provide some feedback.

Also, when we have the opposite case (2014/24 Directive with contracting party/ies as Contracting entities) do the same principles apply?

Regards,
Dean

**Comment by DeanFirkelj** [comment]
Quick update for the "opposite" scenario fresh from the production environment :( - it looks like in case of (2014/24 General Directive having some of the contracting party/ies as Contracting entities) the principle is not the same. 
BR-BT-00010-0022 forces that CA Activity must be provided for the CE.
BR-BT-00610-0022 forces that it's forbidden to have CE Activity at all in the CN for general Directive.

**Reply by YvesJo** [reply, marked as answer]
Hi,
As stipulated in the Directive 25 title:

> Directive 2014/25/EU of the European Parliament and of the Council of 26 February 2014 on procurement **by entities** operating in the water, energy, transport and postal services sectors and repealing Directive 2004/17/EC Text with EEA relevance

these forms are mainly for Entities, therefore for these forms, the Activity Entity (BT-610) is required. 
Some Authorities may also act as Entities, in which case, in addition to the specification of the Activity Entity (BT-610) they are involved in with this procurement, their usual Activity Authority (BT-10) also has to be specified.
Specification of BT-10 and BT-610 has to be done per Buyer (i.e. cac:ContractingParty).
Directive 24 forms only allow BT-10 (Activity Authority). Following Art 1(2) of D24:

> 2.   Procurement within the meaning of this Directive is the acquisition by means of a public contract of works, supplies or services by **one or more contracting authorities** from economic operators chosen by those contracting authorities, whether or not the works, supplies or services are intended for a public purpose.

I hope this helps clarify the rules
KR

**Reply by DeanFirkelj** [reply]
Thanks. Just needed the confirmation that it's by design. 
Regards,
Dean
